tailwind.config.js

```diff
@@ -1,5 +1,6 @@
 /** @type {import('tailwindcss').Config} */
 export default {
+  prefix: 'tw-',
```

Comment:
better solution is to disable preflight for tailwind: https://tailwindcss.com/docs/preflight

Reply:
Yeah I disabled preflight in tailwind.config.js with a coreplugins:{ preflight: false,}, do you mean something else from what I did there?

Reply:
oh is it committed in the tailwind config file? I'm not seeing it I don't think

Reply:
huh, yeah, must be part of the merge conflict, had that change commited weeks ago lol